x/twap/logic.go

```diff
@@ -8,6 +8,9 @@ import (
 	"github.com/osmosis-labs/osmosis/v10/x/twap/types"
 )
 
+// TODO: configure recordHistoryKeepPeriod via parameter.
```

Comment:
by parameter, does it indicate gov params?

Reply:
Exactly: https://github.com/osmosis-labs/osmosis/issues/2371